pydynamo/storage/disk/sstable.py

```diff
@@ -0,0 +1,95 @@
+from ..store import Store
+from ..memory.inmemorystore import InMemoryStore
+from ..error import StorageException
+from ..error import ErrorType
+from datetime import datetime
+from .sstableiterator import SStableIterator
+from struct import pack
+
+
+class SSTable(Store):
+    def __init__(self, store_name: str, store_id: int,
+                 mem_table: InMemoryStore, percentage: float) -> None:
+        self.store_name = store_name
+        self.sstable_file_name = store_name + str(store_id) + ".db"
+        self.index_file_name = store_name + str(store_id) + ".index"
+        self.mem_table = mem_table
+        self.percentage = percentage
+        # index_table:
+        # row: (key, offset)
+        self.index_table: list = []
+        self.size = 0
+        self.int_size = 4
```

Comment:
for this constant you should make it class constant with UPPER_CASE https://stackoverflow.com/questions/2682745/how-do-i-create-a-constant-in-python